src/script/lua_api/l_env.cpp

```diff
@@ -991,16 +971,28 @@ int ModApiEnvMod::l_find_nodes_in_area_under_air(lua_State *L)
 	std::vector<content_t> filter;
 	collectNodeIds(L, 3, ndef, filter);
 
+	bool grouped = lua_isboolean(L, 4) && readParam<bool>(L, 4);
+
+	auto iterate = [&] (auto callback) {
```

Comment:
No need to copy.
```suggestion
	auto iterate = [&] (auto &&callback) {
```

Reply:
@Desour Not copy what? It's a function pointer.

Reply:
It's a lambda.
Not sure how bad the copying is if it's just capturing by reference, but still, there's no need to take the lambda by value.

Reply:
FYI: I didn't opt for the potentially simpler (and type-checked) solution using `std::function` due to performance concerns